source/actors/player/PlayerHealth.hx

```diff
@@ -0,0 +1,141 @@
+package actors.player;
+
+import flixel.group.FlxGroup;
+
+class PlayerHealth {
```

Comment:
I like the idea making this inherit from FlxTypedGroup<T> instead of containing an FlxTypedGroup - I think it'll make it a bit simpler to work with. With that, we can just add this to the active state directly instead of having to get in and access a specific public property of it.